src/styles/blocks/main-data.scss

```diff
@@ -0,0 +1,9 @@
+@import "../utils/variables";
+
+.main-data {
+  display: flex;
+  flex-wrap: wrap;
+  max-inline-size: $rowWidth;
+  margin: auto;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.